cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -299,6 +298,9 @@
      */
     public static final String PROGRAM_STATUS_RECORD_EVENT_TOPIC = "program.status.record.event.topic";
 
+    //Holds a comma separated list of enabled capabilities
```

Comment:
Use javadoc style comment for public field `/** ... */`

Reply:
Done